assets/src/application/ui/layout/entityList/pagination/usePaginationConfig.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External imports
+ */
+import { useMemo } from '@wordpress/element';
+import { _x } from '@eventespresso/i18n';
+
+/**
+ * tracks current page for any paginated list
+ *
+ * @param {Object} paginationConfig
+ * @return {Object} - pagination config
+ */
+const usePaginationConfig = (paginationConfig) =>
+	useMemo(() => {
+		const defaultConfig = {
+			labels: {
+				first: _x('First', 'First thing in list', 'event_espresso'),
+				last: _x('Last', 'Last thing in list', 'event_espresso'),
+				prev: _x('Prev', 'Previous thing in list', 'event_espresso'),
+				next: _x('Next', 'Next thing in list', 'event_espresso'),
```

Comment:
Is this being used still?
More importantly, can this even be changed on the AntD pagination component?
I'm not seeing that in their API.

If we are not or can not use this, then might as well get rid of it

Reply:
Got rid of that.
Was hesitant on throwing away that, thought that maybe there is a possibility to extract some ~~juice~~ code